tests/js/common/aql/aql-arangosearch-inverted-index-search-alias-community.js

```diff
@@ -952,12 +954,12 @@ function IResearchInvertedIndexSearchAliasAqlTestSuiteCommunity() {
                     ]
                 }
             ];
-            for (let i = 0; i < failDefinitions.length; i++) {
-                let d = failDefinitions[i];
-                let viewName = `failed${i}`;
+            for (let i = 1; i <= failDefinitions.length; i++) {
+                let d = failDefinitions[i - 1];
+                let viewName = `failed${i - 1}`;
                 try {
                     db._createView(viewName, 'search-alias', d);
-                    assertTrue(false);
+                    assertFalse(i);
```

Comment:
Maybe this would be more clear?
```suggestion
                    fail();
```

Reply:
But with fail we cannot get iteration of fail